yuhow/pstake.py

```diff
@@ -1,806 +0,0 @@
-#!/usr/bin/env python
```

Comment:
The reason I use PSTricks is it allows us to keep the plain-text source files of the schematics.  It's OK to store binary files like pptx, eps, or pdf in the repository, but they either require a dedicated application or cannot be easily edited.  Could you elaborate why you choose to not use PSTricks?